pyproject.toml

```diff
@@ -87,40 +87,34 @@ Issues = "https://github.com/materialsproject/pymatgen/issues"
 Pypi = "https://pypi.org/project/pymatgen"
 
 [project.optional-dependencies]
+abinit = ["netcdf4>=1.6.5"]
 ase = ["ase>=3.23.0"]
-# tblite only support Python 3.12+ through conda-forge
-# https://github.com/tblite/tblite/issues/175
-tblite = ["tblite[ase]>=0.3.0; python_version<'3.12'"]
-vis = ["vtk>=6.0.0"]
-abinit = ["netcdf4>=1.7.1"]
```

Comment:
this separate `abinit` extra and the `optional` extra requires different versions of `netcdf4`: https://github.com/materialsproject/pymatgen/blob/aa37e99fa67a0e9f996b645be26c12a8f42a4a98/pyproject.toml#L113

pin `netcdf4>=1.7.1` might lead to CI failure in Ubuntu (which I cannot recreate locally)